code/datums/gamemode/role/rambler.dm

```diff
@@ -5,6 +5,8 @@
 	special_role = RAMBLER
 	logo_state = "rambler-logo"
 	wikiroute = ROLE_MINOR
+	default_admin_voice = "Xavier"
```

Comment:
Chief Master Guru is the name of Xavier's mentor, an Appachistani mystic

Reply:
I've shamefully only watched like 2 episodes, thank you.